apps/src/pythonlab/layout/HorizontalLayout.tsx

```diff
@@ -11,8 +11,8 @@ import ResizeBar, {
 
 import moduleStyles from './layout.module.scss';
 
-const MIN_RIGHT_PANEL_WIDTH = 500;
-const MIN_LEFT_PANEL_WIDTH = 200;
+const MIN_RIGHT_PANEL_WIDTH = 300;
```

Comment:
Curious how you decided on these numbers and those in `VerticalLayout`.

Reply:
It was arbitrary, it was based on what felt "too small" when playing around with sizes.